client/components/grade-table.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-useless-constructor */
+
+import React from 'react';
+import Grade from './grade';
+
+class GradeTable extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <table className="table">
+        <thead className="table-dark">
+          <tr>
+            <th>Student Name</th>
+            <th>Course</th>
+            <th>Grade</th>
+          </tr>
+        </thead>
+        <tbody>
+          {this.props.grades
```

Comment:
Arrays are always truthy, so you want to check if the length of the array `=== 0` instead.